deployment/helm-repo/cloud-init/development.yaml

```diff
@@ -130,7 +130,7 @@ write_files:
       echo "Helm installed!"
 
       echo "Adding helm repo..."
-      if ! helm repo add united-manufacturing-hub https://repo.umh.app/; then
+      if ! helm repo add united-manufacturing-hub https://0-9-10-hf1.united-manufacturing-hub.pages.dev/; then
```

Comment:
:)

Reply:
It's still in testing (therefor also tagged as WIP).
I can't to draft PR, else CF won't deploy